config/environments/production.rb

```diff
@@ -14,17 +14,30 @@
   config.consider_all_requests_local       = false
   config.action_controller.perform_caching = true
 
-  # Enable Rack::Cache to put a simple HTTP cache in front of your application
-  # Add `rack-cache` to your Gemfile before enabling this.
-  # For large-scale production use, consider using a caching reverse proxy like nginx, varnish or squid.
-  # config.action_dispatch.rack_cache = true
+   # Ensures that a master key has been made available in either ENV["RAILS_MASTER_KEY"]
+  # or in config/master.key. This key is used to decrypt credentials (and other encrypted files).
+  # config.require_master_key = true
```

Comment:
should we be using this?

Reply:
We can adopt this next in a separate PR, get rid of secrets.yml file and make use of config/credentials.yml.enc
This master key will be the one to encrypt and decrypt the credentials.yml.enc

Reply:
ok yes, separate PR another time. thanks